fastlane/lib/fastlane/actions/update_project_code_signing.rb

```diff
@@ -39,8 +40,12 @@ def self.available_options
                                        end),
           FastlaneCore::ConfigItem.new(key: :udid,
                                        env_name: "FL_PROJECT_SIGNING_UDID",
-                                       description: "The UDID of the provisioning profile you want to use",
-                                       default_value: ENV["SIGH_UDID"])
+                                       description: "DEPRECATED: see :uuid",
+                                       default_value: ENV["SIGH_UUID"]),
```

Comment:
I changed this to UUID, because I believe the intention is for this to default to the result of the sigh action, as opposed to defaulting the Environment variable set by the user.  I could be wrong and happy to revert this line back to UDID.